02-OOP-Inheritance/Ex05/src/classEventEmitter.js

```diff
@@ -0,0 +1,32 @@
+class eventEmitter {
+	constructor(events) {
+		this.events (eventName, callbacks) = {
```

Comment:
Is better if you create an object. The key for each object would be the event identifier, then the value associated to the key, the set of callbacks. 

```
constructor () {
    this.events = {};
};
```